extensions/src/AWSSDK.Extensions.Bedrock.MEAI/BedrockEmbeddingGenerator.cs

```diff
@@ -116,7 +116,11 @@ public async Task<GeneratedEmbeddings<Embedding<float>>> GenerateAsync(
 
         if (totaltokens is not null)
         {
-            embeddings.Usage = new() { InputTokenCount = totaltokens.Value };
+            embeddings.Usage = new() 
+            {
+                InputTokenCount = totaltokens.Value,
```

Comment:
I'm running an internal build, but just had a quick question: Is using the same value for `InputTokenCount` and `TotalTokenCount` intentional?

Reply:
Thanks, @dscpinheiro.

> Is using the same value for InputTokenCount and TotalTokenCount intentional?

It is. Typically in embedding systems, there's no separate output token count or other tokens to be included in an overall total, so the total token count is just equal to the input token count. Is that not the case with these in bedrock? Happy to change it if my assumption is incorrect, I just couldn't find any other token counts in the output that would suggest otherwise.